### ISSUE TITLE
Jenkins X section not populating at all

### ISSUE THREAD

**Issue by adamsmith448** [jenkins-x/vscode-jx-tools#36]
I installed the jx-tools plugin (v0.0.33) via the VS Code marketplace. The 'Jenkins X' section appears in the file explorer sidebar but is not being populated at all:
![image](https://user-images.githubusercontent.com/28056373/41146821-832b6450-6afc-11e8-9a05-db29b7395ad5.png)
When I hit 'refresh' I get an error message
`command 'PipelineExplorer.refresh' not found`
![image](https://user-images.githubusercontent.com/28056373/41146845-9ccecdca-6afc-11e8-8c99-5d4e94e04ec7.png)

**Comment by adamsmith448** [comment]
If I open VS Code from the command line via `code` then the plugin *does* seem to work. This is running `C:\Program Files\Microsoft VS Code\bin\code.cmd` which looks like the following:
```@echo off
setlocal
set VSCODE_DEV=
set ELECTRON_RUN_AS_NODE=1
call "%~dp0..\Code.exe" "%~dp0..\resources\app\out\cli.js" %*
endlocal```

However, if I just run `C:\Program Files\Microsoft VS Code\Code.exe` then I am having the same problem.

**Comment by rawlingsj** [comment]
I've got a feeling @jstrachan had issues lately with a token not refreshing or something?

**Comment by MrFixit96** [comment]
Did this ever get solved? I am using VS Code on Windows 10 and just installed the plugin via the marketplace and am having this same issue now.

**Comment by jstrachan** [comment]
the plugin needs to give better output for sure. Folks just hit this issue... https://github.com/jenkins-x/vscode-jx-tools/issues/40

so in general I think today the plugin assumes `~/.kube/config` is setup and points at valid config so that `kubectl get node` will return meaningful data. We need a better UX if thats not the case

**Comment by pablog12** [comment]
So is there a way to solve it? 
I have the same issue.

**Comment by jstrachan** [comment]
@pablog12 @MrFixit96 do you have a Jenkins X installation such that `kubectl get node` works ok? Or did you just install the plugin first before your machine could connect to Jenkins X? I think the plugin needs to deal with detecting there are no kubernetes clusters in `~/.kube/config` and displaying some help to guide the user to connect/install Jenkins X?

**Comment by MrFixit96** [comment]
Yes, I did it as you mentioned... I installed the plug-in first. Once I got a working kubectl and a working JenkinsX I reinstalled it and it's now working as desired.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by pablog12** [comment]
@jstrachan I installed it after. Also I did a reinstall and it still does not work.

My cluster is running on IBM KS, the default method they provide to config Kubectl to access it via KUBECONFIG ambient var, the file is an output of ibmcloud CLI.

Something like:
`export KUBECONFIG=/Users/pablo/.bluemix/plugins/container-service/clusters/Drogon/kube-config-sao01-Drogon.yml`

So, I guess the plugin can't find `~/.kube/config` because it does not exists.

How can I create that file? or what can I do to fix it.

Also this is gonna be a problem for anyone using IBM.

**Comment by jstrachan** [comment]
@pablog12  I guess in your case we need to make sure the VS Code extension looks for `$KUBECONFIG` and uses that file instead of `~/.kube/config`.

For now a workaround is:
```
mkdir -p ~/.kube
cp KUBECONFIG ~/.kube./config
```
